server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -125,6 +128,39 @@ public void setupEach() throws IOException {
     userController = new UserController(db);
   }
 
+  /**
+   * Construct an instance of `ContextUtil`, which is essentially
+   * a mock context in Javalin. See `mockContext(String, Map)` for
+   * more details.
+   */
+  private Context mockContext(String path) {
+    return mockContext(path, Map.of());
+  }
+
+  /**
+   * Construct an instance of `ContextUtil`, which is essentially a mock
+   * context in Javalin. We need to provide a couple of attributes, which is
+   * the fifth argument, which forces us to also provide the (default) value
+   * for the fourth argument. There are two attributes we need to provide:
+   *
+   *   - One is a `JsonMapper` that is used to translate between POJOs and JSON
+   *     objects. This is needed by almost every test.
+   *   - The other is `maxRequestSize`, which is needed for all the ADD requests,
+   *     since `ContextUtil` checks to make sure that the request isn't "too big".
+   *     Those tests fails if you don't provide a value for `maxRequestSize` for
+   *     it to use in those comparisons.
+   */
+  private Context mockContext(String path, Map<String, String> pathParams) {
+    return ContextUtil.init(
+        mockReq, mockRes,
+        path,
+        pathParams,
+        HandlerType.INVALID,
+        Map.ofEntries(
+          entry(JsonMapperKt.JSON_MAPPER_KEY, javalinJackson),
```

Comment:
After changing the import as suggested by @floogulinc, we can just refer directly to `JSON_MAPPER_KEY`.

```suggestion
          entry(JSON_MAPPER_KEY, javalinJackson),
```